packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -182,6 +187,13 @@ contract Deploy is Deployer {
 
     /// @notice Gets the address of the SafeProxyFactory and Safe singleton for use in deploying a new GnosisSafe.
     function _getSafeFactory() internal returns (SafeProxyFactory safeProxyFactory_, Safe safeSingleton_) {
+        if (getAddress("SafeProxyFactory") != address(0)) {
+            // The SafeProxyFactory is already saved, we can just use it.
+            safeProxyFactory_ = SafeProxyFactory(getAddress("SafeProxyFactory"));
+            safeSingleton_ = Safe(getAddress("SafeSingleton"));
+            return (safeProxyFactory_, safeSingleton_);
```

Comment:
Why was this block not needed before but is needed now?

Reply:
Because we're now calling this method twice, which results in two calls to `save("SafeProxyFactory")`, which [reverts](https://github.com/ethereum-optimism/optimism/blob/maur/deploy-council-safe/packages/contracts-bedrock/scripts/Artifacts.s.sol#L182)

Reply:
`_getSafeFactory` is now only called once AFAICT so this could be removed to shrink the diff further?